### PR TITLE
Drop python dateutil

### PR DESCRIPTION
**Pull request by cajun-rat**
Python dateutil isn't packaged for Yocto, which makes this fail when built in meta-updater

### CODE REVIEW

**Comment by patrickvacek** [comment]
We could package dateutil in Yocto, right? Otherwise, I think we need to find some way to do this correctly. If you want to merge this now, I understand, but we should still get it right very soon.

**Comment by cajun-rat** [comment]
I think the string representation used does sort correctly in lexicographic order, so won't the two implementations have the same behaviour?

**Comment by cajun-rat** [comment]
The main reason to get this in is so that we can test the build in Yocto again, though.

**Comment by patrickvacek** [comment]
> I think the string representation used does sort correctly in lexicographic order, so won't the two implementations have the same behaviour?

Unfortunately, no:

```
>>> a = "cli-0.4.0-20-gc1fb488.tgz"
>>> b = "cli-0.4.0-8-g9d6664d.tgz"
>>> a < b
True
```

**Comment by cajun-rat** [comment]
But this is sorting by Last modified date (the first item of the tuple that is the value of the versions dict)?

File: scripts/get_garage_sign.py [hunk 1]
```diff
@@ -52,7 +51,7 @@ def main():
             else:
                 name = name_ext
     else:
-        name = sorted(versions, key=(lambda name: dp.parse(versions[name][0])))[-1]
+        name = sorted(versions, key=(lambda name: (versions[name][0])))[-1]
```
Comment (lbonn) [inline scripts/get_garage_sign.py]: Was there before but `max` should be equivalent and have a clearer intent

Comment (cajun-rat) [reply scripts/get_garage_sign.py]: Good idea. Fixed.

**Comment by cajun-rat** [comment]
@OYTIS you added the dateutil code: is this change going to break something?

**Pull request merged**